scripts/ma_dictionaries.py

```diff
@@ -33,7 +33,7 @@
     "bdi": "Burundi",  # Country
     "bel": "Belgium",  # Country
     "ben": "Benin",  # Country
-    "bes": "Bonaire, Sint Eustatius and Saba",  # Country
+    "bes": "Bonaire, (Sint Eustatius and Saba)",  # Country
```

Comment:
In this case parentheses are not necessary. 
"Bonaire" is not a shorter name of a country.